recipes/alamode/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+mkdir build; cd build
+cmake -DUSE_MKL_FFT=no -DSPGLIB_ROOT=${PREFIX} ..
+make -j${NUM_CPUS}
```

Comment:
Is there a make check/test we can run here?

Reply:
At least I do not see any in https://github.com/ttadano/alamode/blob/develop/alm/Makefile.linux and https://github.com/ttadano/alamode/blob/develop/anphon/Makefile.linux